lib/challenge.js

```diff
@@ -8,7 +8,7 @@ const ATCQueue = function () {
 ATCQueue.prototype.aircraftCount = function () {
 }
 
-ATCQueue.prototype.enqueue = function (/* aircraft */) {
+ATCQueue.prototype.enqueue = function (aircraft) {
```

Comment:
This would have previously caused a linter error. If it no longer does, than it's fine.